spec/integration/client_spec.rb

```diff
@@ -1,8 +1,5 @@
 require "spec_helper"
 
-java_import java.util.logging.Logger
-java_import java.util.logging.Level
```

Comment:
Yes, all specs use spec_helper, so I don't think it's needed, we will just be repeating ourselves